src/relay/blockchain/identity_events.py

```diff
@@ -0,0 +1,26 @@
+from .events import BlockchainEvent
+
+FeePaymentEventType = "FeePayment"
+
+
+class FeePaymentEvent(BlockchainEvent):
+    @property
+    def value(self):
+        return self._web3_event.get("args").get("value")
+
+    @property
+    def from_(self):
```

Comment:
Not part of the review, but could anybody point me to a document or explain
me himself what the convention about the `_` usage is? I only know these:
- in Python a leading underscore signals the intended module scope
- in Solidity a leading underscore signals a local function parameter
- in Solidity a following underscore is used to avoid name collisions

So how do we use it here?

Reply:
You can't name a parameter `from` in python because it means a `from` loop.
If you would want to do that, you'd write `from = 'whatever'` and python would think you start a `from` loop and then not expect the following `=` sign.